apps/backend/src/modules/users/user.entity.ts

```diff
@@ -199,9 +200,7 @@ class UserEntity implements Entity {
 					? true
 					: false,
 			hasAnsweredQuizQuestions:
-				this.onboardingAnswers && this.onboardingAnswers.length > ZERO_INDEX
-					? true
-					: false,
+				this.quizAnswers && this.quizAnswers.length > ZERO_INDEX ? true : false,
```

Comment:
I think these operators already return a `boolean` value, right?

Reply:
I want this to return purely boolean and not NULL so I did that.

Reply:
I think then you can wrap it in `Boolean()`